docs/source/conf.py

```diff
@@ -34,9 +36,7 @@
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
-# TODO: There must be a way to import this from `setup.py` so we don't have to
-# update it manually for each release.
-version = "9.7.2.dev0"
+version = release = version("plone.restapi")
```

Comment:
When I last tried, I couldn't get this to work. I guess your work with replacing `pkg_resources` elsewhere now offers support?

Reply:
@stevepiercy Unlike other similar errors recently, you weren't seeing this because of a setuptools problem. You were seeing it because you were running it without installing `plone.restapi`. I fixed it by making sure the make command to build the docs installs plone.restapi too.